docs/_articles/en/user-guide/development-models.md

```diff
@@ -118,7 +118,7 @@ You can delete source from your project and from your non-source-tracked org.
 To develop in non-scratch orgs and use Org Browser to retrieve source:
 
 1. Open the VS Code editor and from the Command Palette, run **SFDX: Create Project**.
-1. In the Side Bar of the code editor, click Org Picker. This opens the Command Palette and you can select a command to authorize an org, Dev Hub, or create a scratch org; Or you can select from the list of authorized orgs.
+1. In the code editor's status bar, click Org Picker. This opens the Command Palette and you can select a command to authorize an org, Dev Hub, or create a scratch org; Or you can select from the list of authorized orgs.
```

Comment:
Two tiny nits:

- To get around the "This" not having a noun after it, you can reword the second sentence like this: "The Command Palette opens and you can select a command..."  (I'm personally not bothered by it, but Acrolinx complains). 
- I think the semi-colon after "..or create a scratch org;"  should be a period rather than semi-colon.

Reply:
Thanks, I noticed those things but didn't address them. Will do.